Server/Server.Core/Server.Core.Static/ServerBuilder.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <cstddef>
+#include <string>
+#include <map>
+#include <memory>
+
+#include "Server.hpp"
+
+
+namespace Server::Builder
+{
+    class ServerBuilder
+    {
+    public:
+        ServerBuilder() = default;
+        ~ServerBuilder() = default;
+ 
+        inline ServerBuilder& setValue(std::pair<std::string, std::string> keyValue)
+        {
+            _arguments.insert(keyValue);
+            return *this;
+        }
+
+        Server* make()
```

Comment:
Are you sure that's the right title?

Reply:
Typically, used method `make`. If you have any better option, go on, open to suggestions